googlecdn.js

```diff
@@ -13,20 +13,42 @@ var data = {
 
 var bowerUtil = require('./util/bower');
 
+function getBowerName(bowerJson, name) {
```

Comment:
A small thing, but I dislike reassigning parameters. Could we clarify this a bit, e.g.

```
function normalizeBowerName(bowerJson, name) {
    if (bowerJson.dependencies && !bowerJson.dependencies[name]) {
        return name.replace(/\.js$/, '');
    }
    return name;
}```